action.yaml

```diff
@@ -37,15 +41,20 @@ branding:
 runs:
   using: composite
   steps:
-    - name: Run minimum dependency check
+    - name: Run Dependency Test
       run: |
+        echo "Dependency Test strategy: ${{ inputs.strategy }}"
         if [ "${{ inputs.additional-env-vars }}" != "" ]
         then {
           echo -e "${{ inputs.additional-env-vars }}" > /tmp/dotenv.env
           export $(tr '\n' ' ' < /tmp/dotenv.env)
         }
         fi
-        Rscript ${GITHUB_ACTION_PATH}/script.R ${{ inputs.repository-path }} ${{ inputs.build-args }} ${{ inputs.check-args }}
+        Rscript ${GITHUB_ACTION_PATH}/script.R
       shell: bash
       env:
         GITHUB_PAT: "${{ inputs.github-token }}"
+        VERDEPCHECK_REPOSITORY_PATH: ${{ inputs.repository-path }}
```

Comment:
Why we need to pass the inputs as a env var and not as a script argument as previously?

Reply:
Please see my explanation above.